dashboard/utils.py

```diff
@@ -128,11 +128,10 @@ async def _handler_route(*args) -> aiohttp.web.Response:
                     #   * (self, Request)
                     req = args[-1]
                     return await handler(bind_info.instance, req)
-                except Exception as e:
```

Comment:
job_head part makes sense to me, but why is this needed ?

Reply:
Ah I reverted it, but this would get rid of the insane behavior where we return a 200 when we have an exception…

Reply:
We should do it sometime soon but it broke tests